META-INF/com/google/android/update-binary

```diff
@@ -25,19 +25,26 @@ WORK_FOLDER='microg'
 ZIP_FILE="$3"
 ADDOND='70-microg.sh'
 PRIVAPP_PERMISSIONS='privapp-permissions-microg.xml'
+SYSTEM="/system"
 
 cd /tmp
 mkdir $WORK_FOLDER
 cd $WORK_FOLDER
 unzip -o "$ZIP_FILE"
 
 mount /system
-
+# A/B system-as-root devices have the actual system partition
+# mounted under /system/system when in the recovery.
+# Snippet taken from F-Droid Privileged Extension.
+system_as_root=`getprop ro.build.system_root_image`
```

Comment:
Can you change this for the `$(command)` notation? Just to be coherent with the line 47 below.

Reply:
Nevermind, I'll change that, :+1:

Reply:
Indeed, I paid no attention to keep the style coherent with there rest of the code.
Thanks for fixing it!